TechtonicaModLoader/MyClasses/Managers/ModManager.cs

```diff
@@ -16,6 +17,10 @@ public static class ModManager
     {
         // Objects & Variables
 
+        // The release of 1.0, used to filter mods that are likely broken.
+        internal static readonly DateTime techtonicaReleaseDate = new DateTime(2024, 11, 7);
```

Comment:
I released a lot of mine on this date, so test that they're not blocked

Reply:
Already verified and I double checked when I updated the allow list.